docs/getting-started/usage.md

```diff
@@ -14,31 +14,34 @@ The following steps will create a React app that uses Instructure UI. Recommende
 ---
 type: code
 ---
+
 npm create vite@latest my-cool-app
 ```
 
-During the interactive project generation phase, please choose React and either JavaScript or TypeScript as the project language. After you have installed the dependencies by running `npm install`, you can try it by running `npm run dev`. 
+During the interactive project generation phase, please choose React and either JavaScript or TypeScript as the project language. After you have installed the dependencies by running `npm install`, you can try it by running `npm run dev`.
 
-##### Add InstUI dependencies to your `package.json` 
+##### Add InstUI dependencies to your `package.json`
 
 We recommend that you add the `@instructure/ui` meta-package, this contains all of our UI components (substitute the version number with the latest one):
 
 ```md
 ---
 type: code
 ---
+
 {
-    ...
-    "dependencies": {
-    ...
-        "@instructure/ui": "^9"
-    }
+...
+"dependencies": {
+...
+"@instructure/ui": "^10"
+}
 }
 ```
 
 Run `npm install`, so InstUI is downloaded to your `node_modules` folder and can be used.
 
-##### Try the sample code 
+##### Try the sample code
+
 Now you are ready to use InstUI, let's try it out. Replace the code in `App.js` with the following:
```

Comment:
later in this file it talks about the `instructure` theme, can you remove that part?

Reply:
done